apps/web/screens/Article.tsx

```diff
@@ -563,6 +564,7 @@ const ArticleScreen: Screen<ArticleProps> = ({ article, namespace }) => {
             portalRef.current,
           )}
       </SidebarLayout>
+      <OrganizationChatPanel slug={article.organization[0]?.slug} />
```

Comment:
I see that the OrganizationChatPanel checks for the slug `syslumenn`.
Here you are getting the first organization and passing it along.

I decided to map the organization slugs of articles in the importer to see.
Is it ok to hardcode to `organization[0]?.slug` when the importer data on mapped organization slugs looks like this:

```
//excerpt
["syslumenn","thjodskra-islands"]
["syslumenn","tryggingastofnun"]
["syslumadurinn-a-nordurlandi-eystra","syslumenn"]
["syslumadurinn-i-vestmannaeyjum","syslumenn"]
["syslumadurinn-a-hoefudborgarsvaedinu","syslumenn"]
["syslumadurinn-a-hoefudborgarsvaedinu","syslumenn"]
["syslumadurinn-a-hoefudborgarsvaedinu","syslumenn"]
["syslumadurinn-a-hoefudborgarsvaedinu","syslumenn"]
["syslumadurinn-a-hoefudborgarsvaedinu","syslumenn"]
["syslumadurinn-a-hoefudborgarsvaedinu","syslumenn"]
["syslumadurinn-a-hoefudborgarsvaedinu","syslumenn"]
["syslumadurinn-a-hoefudborgarsvaedinu","syslumenn"]
["syslumadurinn-a-austurlandi","syslumenn"]
["tryggingastofnun","syslumenn"]
["syslumadurinn-a-nordurlandi-vestra","syslumenn"]
["syslumadurinn-a-nordurlandi-vestra","syslumenn"]
["syslumadurinn-a-sudurlandi","syslumenn"]
["syslumadurinn-a-sudurlandi","syslumenn"]
["syslumenn","thjodskra-islands"]
```

There are examples of `syslumenn` being at organization[1].